### PR TITLE
Fastlane setup for tagging, releasing, and internal testing

### PR DESCRIPTION
**Pull request by kueda**
Instead of separate Fastlane configurations for iOS and Android, this attempts to make one configuration that does both. In this PR it only goes as far as pushing for internal testing since that's all we can do with the draft app we're using at the moment. This should at least make it easier to push test releases.

The README and example config files should provide enough info to get set up, but I'm sure I've forgotten some bits. In general, the release process should look like this:

```zsh
fastlane tag
fastlane release
fastlane internal
```

That will require you to enter release notes, create a github tag, create a github release, and push to the Google Play internal test track and TestFlight.

### CODE REVIEW

**Review by albullington — COMMENTED** [review]
Ok, tried setting this up and got stuck on `fastlane tag` as we chatted about in Slack. Wasn't able to create a tag, and therefore wasn't able to use `fastlane release` or create a build. This all looks good to me in theory.

File: package.json [hunk 1]
```diff
@@ -38,21 +38,21 @@
     "radio-buttons-react-native": "^1.0.4",
     "react": "17.0.2",
     "react-i18next": "^11.16.1",
-    "react-native": "^0.67.4",
+    "react-native": "^0.68.2",
```
[the remark this answers is not in the document]
Comment (kueda) [reply package.json:41]: Well, this was one of my build problems... that I guess I wasn't hitting when I was testing release builds (though I thought I tried a debug one too). Thanks!

**Comment by albullington** [comment]
Ok, I was able to successfully create tags and a github release. I wasn't able to use `fastlane internal` to create a new build -- the error I got says my AAB file is missing, which makes me think I have the wrong values in `keystore.properties`.

We might want to do some code cleanup to avoid future confusion -- it looks like there are still be `ios/fastlane` and `android/fastlane` folders in addition to the main `fastlane` folder. We no longer need `INATRN_RELEASE_STORE_FILE` or `INATRN_RELEASE_KEY_ALIAS` values in `android/gradle.properties`. And I think we can remove the `getPassword` and `pass` definitions from `android/app/build.gradle` now that we're using `keystoreProperties` instead.

File: fastlane/Fastfile [hunk 2]
```diff
@@ -0,0 +1,297 @@
+require "fileutils"
```
[the remark this answers is not in the document]
Comment (kueda) [reply fastlane/Fastfile:1]: I'm hesitant to do that b/c I feel like automatic updates often lead to unexpected failures, like with homebrew, so maybe let's just continue with manual updates and see how it feels.

**Pull request merged**